server/server.js

```diff
@@ -13,6 +18,16 @@ app.get('/',(req, res) =>{
   res.status(200).send('hello world et tonton');
 })
 
-app.post('/',(req, res) =>{
-  res.status(200).send('hello world et tonton');
-})
\ No newline at end of file
+app.post('/boxes', (req, res) => {
+
+  const formData = req.body;
```

Comment:
you should check if formData effectively contains the data you need. and take a look at 4xx status codes ;)

https://en.m.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors